converter/src/main/java/gov/cms/qpp/conversion/validate/CpcQualityMeasureSectionValidator.java

```diff
@@ -76,8 +76,8 @@ private void verifyOverallCount(Checker checker) {
 	 * A holder of CPC+ group specific configuration information.
 	 */
 	enum CpcGroupMinimum {
-		A("outcome", 2),
-		B("complex process", 2);
+		Outcome_Measure("outcome", 2),
+		Other_Measure("complex process", 2);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)